src/histogram.rs

```diff
@@ -306,6 +308,11 @@ impl Histogram {
     pub fn start_timer(&self) -> HistogramTimer {
         HistogramTimer::new(self.clone())
     }
+
+    /// `local` returns a `LocalHistogram` for single thread usage.
+    pub fn local(&self) -> LocalHistogram {
+        LocalHistogram::new(self.clone())
+    }
```

Comment:
Will it be a problem if other thread keeps using the old `Histogram` after creating the `LocalHistogram`?